docs/modules/ROOT/pages/references.adoc

```diff
@@ -1,98 +1,115 @@
+. [[ref:20]]SEMIC Style Guide for Drafting Semantic Specifications. SEMIC. Available at: https://github.com/SEMICeu/Draft_guidelines_2020 (Accessed: January 3, 2023).
+. [[ref:21]] Miles, A. and Bechhofer, S. (eds) (2019) SKOS Simple Knowledge Organization System Reference. Available at: https://www.w3.org/TR/skos-reference/.
+. [[ref:22]] Costetchi, E. (2020) eProcurement UML conceptual model conventions, sec 3.1. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf.
+. [[ref:23]] Costetchi, E. (2020) eProcurement UML conceptual model conventions, sec 3.6. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf.
+. [[ref:24]] Costetchi, E. (2020) eProcurement UML conceptual model conventions, sec 3.7. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf.
+. [[ref:25]] hStani, E. Principles for creating good definitions_v1.00. Available at: https://github.com/SEMICeu/OpenGuidelines/blob/main/Principles%20for%20creating%20good%20definitions_v1.00.pdf.
+. [[ref:26]] Costetchi, E. (2020) eProcurement UML conceptual model conventions, sec 4.2. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf.
+. [[ref:27]] Costetchi, E. (2020) eProcurement UML conceptual model conventions, sec 4.1. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf.
+. [[ref:28]] object management group (2014) Ontology definition metamodel . OMG. Available at: https://www.omg.org/spec/ODM/1.1/PDF (Accessed: January 2, 2023).
+. [[ref:29]]  SEMICEU-toolchain-manual. SEMICEU. Available at: https://github.com/SEMICeu/toolchain-manual (Accessed: January 3, 2023).
+. [[ref:30]] Costetchi, E. (2020) eProcurement UML conceptual model conventions. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf (Accessed: January 2, 2023).
+. [[ref:31]] Costetchi, E. (2020) Transformation of the eProcurement UML model into a formal OWL ontology. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml2owl-transformation/uml2owl-transformation.pdf.
+. [[ref:32]] P. Archer, S. Goedertier, and N. Loutas. D7.1.3 - study on persistent uris, with
+identification of best practices and recommendations on the topic for the mss
+and the ec. Deliverable, ISA programme of the European Commission, 2012
+. [[ref:33]] D. Wood, R. Cyganiak, and M. Lanthaler. RDF 1.1 concepts
+and abstract syntax. W3C recommendation, W3C, Feb. 2014.
+http://www.w3.org/TR/2014/REC-rdf11-concepts-20140225/.
+. [[ref:34]] T. Bray, M. Sperberg-McQueen, E. Maler, F. Yergeau, and J. Paoli. Extensible
+markup language (XML) 1.0 (fifth edition). W3C recommendation, W3C, Nov.
+2008. http://www.w3.org/TR/2008/REC-xml-20081126/.
```

Comment:
"available at / URL:"

Reply:
fixed (the same because od existing references)